pkg/util/util.go

```diff
@@ -460,3 +462,18 @@ func HandleArrayFormat(name string) string {
 	}
 	return name
 }
+
+func StringBytesToTerabytes(s string) (float64, error) {
+	if s == "" {
+		return 0, nil
+	}
+	bytes, err := strconv.ParseInt(s, 10, 64)
+	if err != nil {
+		return 0, err
+	}
+	return float64(bytes) / bytesInTB, nil
+}
+
+func ConvertStringToFloat64(s string) (float64, error) {
```

Comment:
We should use strconv.ParseFloat(s, 64) instead of ConvertStringToFloat64(s). This can be removed